README.md

```diff
@@ -69,3 +69,22 @@ You can customize the prompt segment behavior by overriding these variables:
 |--------------------------------|---------|-----------------------------------------------------------------------------|
 | `AWS_VAULT_PL_CHAR`            | ☁       | The character to display when logged into an aws-vault profile              |
 | `AWS_VAULT_PL_DEFAULT_PROFILE` | default | Only show the character when logged into this profile, not the profile name |
+
+### avli - login in private browsing window
+
+> This alias is currently only supported in OSX.
+
+This alias will open a new browser window after getting the temporary login URL for your profile.
+
+You can specify a specific browser to handle your login URL by setting `AWS_VAULT_PL_BROWSER` to the bundle name of the
```

Comment:
I wonder if there's a way to make it clearer what the options are for this variable. When I was first reading this, I didn't realize that `AWS_VAULT_PL_BROWSER` needed to be `org.mozilla.firefox` or `com.google.chrome`.

I wonder if a little markdown table (like the one above) could help make that clearer vs. the two subheadings.

Thoughts?

Reply:
🙆‍♂️